apps/web/src/pages/ApplicantDashboardPage/components/FunctionalCommunityListItem.tsx

```diff
@@ -0,0 +1,196 @@
+import { JSX } from "react";
+import { useIntl } from "react-intl";
+import BriefcaseIcon from "@heroicons/react/20/solid/BriefcaseIcon";
+import PresentationChartLineIcon from "@heroicons/react/20/solid/PresentationChartLineIcon";
+import XCircleIcon from "@heroicons/react/20/solid/XCircleIcon";
+import ExclamationTriangleIcon from "@heroicons/react/20/solid/ExclamationTriangleIcon";
+
+import {
+  graphql,
+  PreviewListItemFunctionalCommunityFragment as PreviewListItemFunctionalCommunityFragmentType,
+} from "@gc-digital-talent/graphql";
+import { HeadingLevel, PreviewList } from "@gc-digital-talent/ui";
+
+export const PreviewListItemFunctionalCommunity_Fragment = graphql(
+  /* GraphQL */ `
+    fragment PreviewListItemFunctionalCommunity on CommunityInterest {
+      id
+      jobInterest
+      trainingInterest
+      community {
+        id
+        name {
+          localized
+        }
+        description {
+          localized
+        }
+      }
+    }
+  `,
+);
+
+interface FunctionalCommunityListItemProps {
+  headingAs?: HeadingLevel;
+  functionalCommunityListItemFragment: PreviewListItemFunctionalCommunityFragmentType;
+}
+
+const FunctionalCommunityListItem = ({
+  headingAs,
+  functionalCommunityListItemFragment,
+}: FunctionalCommunityListItemProps) => {
+  const intl = useIntl();
+
+  const sharedIconStyling = {
+    "data-h2-height": "base(x1)",
+    "data-h2-width": "base(x1)",
+    "data-h2-display": "base(inline-block)",
+    "data-h2-vertical-align": "base(bottom)",
+    "data-h2-margin-right": "base(x.25)",
+    "data-h2-padding-top": "base(x.125)",
+  };
+
+  const interestedWork = (
+    <span>
+      <BriefcaseIcon
+        data-h2-color="base(success)"
+        {...sharedIconStyling}
+      ></BriefcaseIcon>
+      {intl.formatMessage({
+        defaultMessage: "Interested in work",
+        id: "1VKNrs",
+        description: "Phrase marking interest in community work opportunities",
+      })}
+    </span>
+  );
+  const notInterestedWork = (
+    <span>
+      <XCircleIcon
+        data-h2-color="base(gray.lighter)"
+        {...sharedIconStyling}
+      ></XCircleIcon>
+      {intl.formatMessage({
+        defaultMessage: "Not interested in work",
+        id: "VDVRPt",
+        description:
+          "Phrase marking lack of interest in community work opportunities",
+      })}
+    </span>
+  );
+  const missingWork = (
+    <span>
+      <ExclamationTriangleIcon
+        data-h2-color="base(error)"
+        {...sharedIconStyling}
+      ></ExclamationTriangleIcon>
+      {intl.formatMessage({
+        defaultMessage: "<red>Missing work info</red>",
```

Comment:
Consider removing the <red> formatting from the string and doing it on the `<p>` element instead so the string can be reused.

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/commit/9368ed0dab4b494a18a55b8fefc2ac88e899f0b8